codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2ConnectionHandlerBuilder.java

```diff
@@ -486,6 +487,24 @@ protected boolean decoupleCloseAndGoAway() {
         return decoupleCloseAndGoAway;
     }
 
+    /**
+     * Determine if the {@code PREFACE} should be automatically flushed when the {@link Channel} becomes active or not.
```

Comment:
Some suggestions:

```
Determine if the <a href="https://datatracker.ietf.org/doc/html/rfc7540#section-3.5">Preface</> should be automatically flushed when the {@link Channel} becomes active or not.
<p>
Client may choose to opt-out from this automatic behavior and manage flush manually if it's ready to send request frames immediately after the preface. It may help to avoud unnecessary latency.

@param flushPreface {@code true} to automatically flush, {@code false otherwise}.
@return {@code this}.
@see <a href="https://datatracker.ietf.org/doc/html/rfc7540#section-3.5">HTTP/2 Connection Preface</>
```